users/tests/test_users.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from users.models import Role, User
+from django.contrib.auth.models import User as DjangoUser
+from falcon.test_support import Random
+from django.core.exceptions import ValidationError
+
+
+@pytest.mark.django_db
+class TestUsers:
+
+    def test_create_user(self, employee_1):
+        assert isinstance(employee_1, User)
+        assert User.objects.filter(user=employee_1.user).exists()
+
+    def test_delete_user(self, employee_1):
+        userId = employee_1.user.id
+        employee_1.delete()
+        assert not User.objects.filter(user=userId).exists()
+        assert not DjangoUser.objects.filter(pk=userId).exists()
+
+    def test_create_user_without_email(self, team_1):
+        with pytest.raises(Exception):
+            User.create_user(
+                username="user1",
+                password="password",
+                first_name="first_name",
+                last_name="last_name",
+                role=Role.EMPLOYEE,
+                team=team_1)
+
+    def test_create_user_with_long_first_name(self, team_1):
```

Comment:
if you will use parameterize it will be easier to add the following test:
test_create_user_with_long_last_name
test_create_user_with_special_chars_first_name
test_create_user_with_special_chars_last_name

Reply:
same ^^

Reply:
test_create_user_with_special_chars_first_name
test_create_user_with_special_chars_last_name

Are not that relevant, see [here](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/)